source/lmp/pair_deepmd.cpp

```diff
@@ -354,8 +354,10 @@ PairDeepMD::PairDeepMD(LAMMPS *lmp)
                "Pair deepmd requires metal or real unit, please set it by "
                "\"units metal\" or \"units real\"");
   }
-  ener_unit_cvt_factor =
-      utils::get_conversion_factor(utils::ENERGY, unit_convert);
+  ener_unit_cvt_factor = force->boltz / 8.617343e-5;
```

Comment:
```suggestion
  ener_unit_cvt_factor = force->qe2f;
```

Reply:
I understand why you are suggesting changing to force->qe2f: for "real", it is equal to 23.060549, which is force->boltz / 8.617343e-5. However, there are cases in which force->qe2f does not represent the correct energy conversion factor. For example, for units "si",1 eV = 1.60218e-19 Joule. It can be obtained with 
(force->boltz = 1.3806504e-23) / 8.617343e-5. However, force->qe2f for "si" is 1.0. Thus, I would keep this unchanged.